model/browser/events.yaml

```diff
@@ -0,0 +1,54 @@
+groups:
+  - id: event.browser.web_vital
+    stability: development
+    type: event
+    name: browser.web_vital
+    brief: >
+      This event describes the website performance metrics introduced by Google, See [web vitals](https://web.dev/vitals).
+    body:
+      id: browser.web_vital
+      requirement_level: required
+      stability: development
+      type: map
+      fields:
+        - id: name
+          type: enum
+          members:
+            - id: cls
+              value: 'cls'
+              brief: Cumulative Layout Shift.
```

Comment:
can we add a `note:` here with external link to the `cls` definition? (and same for others)

Reply:
Updated, added the link to brief